packages/core/src/util/ws.ts

```diff
@@ -0,0 +1,5 @@
+import type { Snowflake } from 'discord-api-types/v10';
+
+export function calculateShardId(guildId: Snowflake, shardCount: number) {
```

Comment:
I feel like this belongs in `@discordjs/util` 👀

Reply:
Agreed